client/src/components/EditMyProfile.js

```diff
@@ -0,0 +1,98 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { Formik, Form } from 'formik';
+import { ProfileContext } from '../context/ProfileContext';
+import * as Yup from "yup";
+import FormField from "../constant/FormField";
+import StyledButton from '../constant/StyledButton';
+import avatar from '../assets/icons/avatar.svg';
+import GraduateForm from './GraduateForm';
+
+const EditMyProfile = () => {
+	const { profile, editProfile, manageEdit, edit }  = useContext(ProfileContext);
+
+	const handleSubmit = (values)=>{
+		const { name, email, website, phone } = values;
+		const editedProfile ={
+			'id':profile.id,
+			'full_name':name,
+			'email':email,
+			'website':website,
+			'phone':phone,
+		};
+		editProfile(editedProfile);
+	};
+
+	const handleReset = ()=>{
+		manageEdit(false);
+		console.log('ed', edit);
+	};
+
+	const initialValue = profile ? { name:`${profile.name}`, email:`${profile.email}`, website:`${profile.website}`, phone:`${profile.phone}` }:{ name:'', email:'', website:'', phone:'' };
+
+	return (
+		<Container >
+			<Formik
+				initialValues={initialValue}
```

Comment:
I might be missing something but do you need to create a new `initialValue` variable here? I think you could just use `profile` in there. If `profile` is null, it will just show your placeholders but please double-check.
```suggestion
				initialValues={profile}
```